Apps/Sandcastle/gallery/Labels.html

```diff
@@ -214,11 +215,55 @@
               Sandcastle.highlight(setRightToLeft);
             },
           },
+          {
+            text: "Animate Label",
+            onselect: function () {
+              animateLabel();
+              Sandcastle.highlight(animateLabel);
+            },
+          },
         ]);
 
         Sandcastle.reset = function () {
           viewer.entities.removeAll();
+          if (Cesium.defined(labelListenerCallback)) {
+            labelListenerCallback();
+          }
         };
+
+        function animateLabel() {
+          var entity = viewer.entities.add({
+            position: Cesium.Cartesian3.fromDegrees(-75.1641667, 39.9522222),
+            label: {
+              text: "Philadelphia",
+            },
```

Comment:
Since the `preUpdate` event is setting the outline width it would be good to add the outline style to the label and see the outline get animated as well.

```suggestion
            label: {
              text: "Philadelphia",
              outlineColor: Cesium.Color.BLACK,
              outlineWidth: 0,
              style: Cesium.LabelStyle.FILL_AND_OUTLINE,
            },
```